DQMOffline/RecoB/interface/JetTagPlotter.h

```diff
@@ -80,6 +84,12 @@ class JetTagPlotter : public BaseBTagPlotter {
 
   // reconstructed jet phi
   FlavourHistograms<double> * dJetRecPhi;
+  
+  // jet Phi larger than requested discrimnator cut
+  FlavourHistograms<double> * dJetPhiDiscrCut;
```

Comment:
I agree, please change the data members to `std::unique_ptr<T>`, allocate them with `std::make_unique<T>(args)` and get rid of the `delete`s.